webrtc.html

```diff
@@ -2033,6 +1684,537 @@
+
+          <dd>
+            <p>This event handler, of event handler event type <code><a href=
+            "#event-track">track</a></code>, MUST be fired
+            by all objects implementing the <code><a>RTCPeerConnection</a></code>
+            interface. </p>
+            <p>It is called any time a <code>MediaStreamTrack</code> is added
+            by the remote peer; the process for this is indicated below.</p>
+            <p>Rejection of incoming tracks can be done by the application after
+            receiving the ontrack event, by stopping the track.</p>
+          </dd>
+
+        </dl>
+        <section>
+          <h4>Processing Remote MediaStreamTracks</h4>
+          <p>The word "components" in this context refers to an RTP media flow
+           and does not have anything to do with how [[ICE]] uses the term
+           "component".</p>
+
+          <p>When a user agent has reached the point where a
+          <code><a>MediaStreamTrack</a></code> can be created to represent an incoming
+          component, the user agent MUST run the following steps:</p>
+
+          <ol>
+            <li>
+              <p>Let <var>connection</var> be the
+              <code><a>RTCPeerConnection</a></code> expecting this media.</p>
+            </li>
+
+            <li>
+              <p>Let <var>streams</var> be a list of <code>
+              <a>MediaStream</a></code> objects that the sender indicated the
+              sent <code><a>MediaStreamTrack</a></code> being a part of. This
+              information needed to collect these objects is part of the remote
+              SDP.</p>
+            </li>
+
+            <li>
+              <p>Run the following steps to create a track
+              representing the incoming component:</p>
+
+              <ol>
+                <li>
+                  <p>Create a <code><a>MediaStreamTrack</a></code> object
+                  <var>track</var> to represent the component.</p>
+                </li>
+
+                <li>
+                  <p>Initialize <var>track's</var> <code><a href=
+                  "getusermedia.html#dom-mediastreamtrack-kind">kind</a></code>
+                  attribute to "<code>audio</code>" or "<code>video</code>"
+                  depending on the media type of the incoming component.</p>
+                </li>
+
+                <li>
+                  <p>Initialize <var>track's</var> <code><a href=
+                  "getusermedia.html#dom-mediastreamtrack-id">id</a></code>
+                  attribute to the component track id.</p>
+                </li>
+
+                <li>
+                  <p>Initialize <var>track's</var> <code><a href=
+                  "getusermedia.html#dom-mediastreamtrack-label">label</a></code>
+                  attribute to "<code>remote audio</code>" or "<code>remote
+                  video</code>" depending on the media type of the incoming
+                  component.</p>
```

Comment:
Still seem that setting the label like this is wrong. It should be transported over the SDP